pywbem/_statistics.py

```diff
@@ -0,0 +1,352 @@
+#!/usr/bin/env python
+#
+# (C) Copyright 2003-2007 Hewlett-Packard Development Company, L.P.
```

Comment:
I think for new modules we add, they should not have these copyright statements, but those of the initial author (you)